### DISCUSSION TITLE
teoria relazionale - 9

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#76] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/e61063d6-a3a4-4104-9573-1bf2fb86cadd)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/964d89ef-476b-4f20-93e9-ca4e883e4532)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/27a2e504-fbb4-4cd7-855f-c28832196047)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/647f799b-c8bf-4072-8a40-e898c703ea3b)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/de6815be-378e-4528-8c6f-e9a66abb50ee)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/ca379db3-3b13-4b75-8658-4f74b48ad9db)
